pages/how-to-export-your-whatsapp-chat.vue

```diff
@@ -1,20 +1,136 @@
 <template>
   <v-container>
-    <ExportExplainer class="mt-10" cta="Export your chat as PDF" to="/" />
+    <ExportExplainer class="mt-10" cta="exportAsPDF" to="/" />
     <PdfExample />
     <Content :page="page" />
+    <v-data-table :headers="headers" :items="desserts" class="elevation-1">
+      <template v-slot:item="item">
+        <v-chip dark>
+          {{ item }}
+        </v-chip>
+      </template>
+    </v-data-table>
     <Cta
       to="/"
       class="pt-0 pb-15"
-      title="Analyze your own WhatsApp data"
-      buttonTxt="Analyze my chat"
-      text="Interested about your own chat data? Take deep dive in your data!"
+      title="analyzeYourOwnWhatsapp"
+      buttonTxt="analyzeMyChat"
+      text="interestedAbout"
     />
   </v-container>
 </template>
 
 <script>
 export default {
+  name: "how-to-export-your-whatsapp-chat",
```

Comment:
![image](https://user-images.githubusercontent.com/23440847/114213746-ad1b6d80-9963-11eb-96f7-1cea4819a566.png)

Das wird noch falsch angezeigt

Reply:
i fixed this